Packs/CommonScripts/Scripts/ScheduleGenericPolling/ScheduleGenericPolling_test.py

```diff
@@ -37,3 +37,25 @@ def test_calculate_end_time(value, expected_result):
 def test_is_value_sanitized(value, expected_result):
     result = is_value_sanitized(value)
     assert result == expected_result
+
+
+@pytest.mark.parametrize('command, expected_result',
+                         [
+                             ("1234", (True, None)),
+                             ("additionalPollingCommandArgNames additionalPollingCommandArgNames",
+                              (False, 'The value of additionalPollingCommandArgNames is malformed.')),
+                             ("pollingCommandArgName additionalPollingCommandArgNames", (True, None)),
+                             ("pollingCommandArgName additionalPollingCommandArgValues pollingCommandArgName "
+                              "additionalPollingCommandArgValues",
+                              (False, 'The value of additionalPollingCommandArgValues, pollingCommandArgName is malformed.')),
+                             ("pollingCommandArgName additionalPollingCommandArgValues pollingCommandArgName "
+                              "additionalPollingCommandArgValues",
+                              (False, 'The value of additionalPollingCommandArgValues, pollingCommandArgName is malformed.')),
+                             ("pollingCommand pollingCommandArgName additionalPollingCommandArgValues pollingCommand "
+                              "pollingCommandArgName additionalPollingCommandArgValues",
+                              (False, 'The value of additionalPollingCommandArgValues, pollingCommandArgName, pollingCommand is'
+                               ' malformed.')),
```

Comment:
Add end to end test which calls to main()
1 test which is successful
1 test which fails on malformed input.

Reply:
[Done](https://github.com/demisto/content/pull/32878/commits/da1e9aa08b3e2ddd1518a4c3a366f26aa7fad463).